src/components/App/App.css

```diff
@@ -3,6 +3,13 @@
   padding: 0;
 }
 
+
```

Comment:
Looks like the conflict is that I deleted all the old App.css styling (now that the logos are gone)